### PR TITLE
Adding job check chart in sync to bors

### PR DESCRIPTION
**Pull request by alallema**
- Adding check chart manifest in sync to Bors action and move it with tests

### CODE REVIEW

**Comment by alallema** [comment]
@meilisearch sync-manifest

**Comment by curquiza** [comment]
bors try

**Comment by alallema** [comment]
bors try

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -42,3 +42,21 @@ jobs:
         with:
           command: install
           config: ct.yaml
+
+  check-chart-manifest-in-sync:
+    name: Check in sync
```
Comment (curquiza) [inline .github/workflows/tests.yaml:47]:
```suggestion
    name: check-chart-manifest-in-sync
```

**Comment by curquiza** [comment]
bors try-

**Review by curquiza — APPROVED** [review]
bors merge

**Pull request merged**